cmd/sysinfo/sysinfo.go

```diff
@@ -16,107 +16,17 @@ limitations under the License.
 package sysinfo
 
 import (
-	"runtime"
-
+	"github.com/k0sproject/k0s/pkg/sysinfo"
 	"github.com/spf13/cobra"
-	system "k8s.io/system-validators/validators"
 )
 
 func NewSysinfoCmd() *cobra.Command {
 	return &cobra.Command{
 		Use:   "sysinfo",
 		Short: "Display system information",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runSysinfo()
+			sysinfo.ReportSysinfo()
```

Comment:
IMO we could handle the error here too by logging it. In case the command fails to read kernel mod info, the output will be only partial and user does not know if everything is ok or not.

Reply:
This needs some consideration. The system-validators library will return unfulfilled non-optional requirements both via the `Reporter` interface (marked as "bad") and as error, whereas "real" errors are solely reported as errors. As far as I can tell, there's no sustainable way to distinguishing those error types. When logging the errors here, we'd have all the failed requirements doubled.

That's probably the reason as to why the previous implementation just disregarded any errors (returning `nil` in any case).